frontend/src/locales/en-US.json

```diff
@@ -424,6 +424,11 @@
       "description": "Allow developers to execute queries for the following databases in the protected environments from SQL Editor.",
       "add-rule": "Add rule",
       "remove-policy-tips": "Remove this rule?"
+    },
+    "release": {
+      "new-version-available": "New version available!",
+      "new-version-content": "The new version {tag} is available now! Please go to the GitHub to check the latest release.",
```

Comment:
We can point them to bytebase.com/docs for upgrades. Btw, shall we show this banner for DBA/Owner only?

Reply:
I think it's ok to show this message to all users? No need to use RBAC here. WDYT @candy2255

Reply:
Only owner/DBA can upgrade Bytebase so that we can annoy less to developers. Developers may not care about upgrade either.

Reply:
I think it is nice to show the change log to all users. Since users can get more information about Bytebase, such as new features, new press.
<img width="971" alt="CleanShot 2022-12-15 at 13 51 23@2x" src="https://user-images.githubusercontent.com/104962472/207783011-3f691ff7-ee02-4901-991b-35afef0a588f.png">

Reply:
Right, but they can share the knowledge and it's a motivation to upgrade. Developers can also check the "Don't show this till next version" to ignore the message.

I think as most of our users are in the FREE plan, they will always see this upgrade modal.
As for the TEAM of ENTERPRISE plan, maybe we need some feedback for this (users' complaints & distribution for different roles and usage frequency)

Reply:
Updated the URL and wording
<img width="674" alt="CleanShot 2022-12-15 at 14 02 49@2x" src="https://user-images.githubusercontent.com/10706318/207784838-613ceecd-a752-480c-8fa0-a408fbf92c25.png">

Reply:
Sure, we can start with all users.